cmd/collector/app/handler/otlp_receiver.go

```diff
@@ -44,7 +44,7 @@ var _ component.Host = (*otelHost)(nil) // API check
 // StartOTLPReceiver starts OpenTelemetry OTLP receiver listening on gRPC and HTTP ports.
 func StartOTLPReceiver(options *flags.CollectorOptions, logger *zap.Logger, spanProcessor processor.SpanProcessor, tm *tenancy.Manager) (receiver.Traces, error) {
 	otlpFactory := otlpreceiver.NewFactory()
-	return startOTLPReceiver(
+	return start_OTLP_Receiver(
```

Comment:
This is not what the linter was complaining about. Your names violate Go coding practices.

Reply:
so how should i fix this naming  problem (Warns on methods with names that differ only by capitalization) ?

Reply:
will it be okay if i name it like `startOTLPReceiverInternal`

Reply:
I think it is fine to use startOTLPReceiver as of camelCase which is generally used in Go.

Reply:
`startOTLPReceiverInternal` seems ok, but this needs to be decided on a case by case basis, there may be better options in other situations.